client/src/component/ticketForm.test.js

```diff
@@ -0,0 +1,20 @@
+import { render, screen, fireEvent} from '@testing-library/react';
+import TicketForm from './ticketForm';
+
+
+test('renders TicketForm', () => {
+  render(<TicketForm />);
+  const firstNameInput = screen.getByLabelText('First Name')
+  expect(firstNameInput).toBeInTheDocument();
+
+});
+
+test('callApi is called when button is clicked', () => {
+    const fn = jest.fn();
```

Comment:
You are right in your comment that this part isn't tested. One might argue that ensure a button click calls an `onClick` is overkill. It would require some code restructuring. This test is still valid by itself if you remove this line.